docs/manual/reference/csv-format.md

```
@@ -0,0 +1,54 @@
+# CSV format reference
+
+This page describes the format of `.csv` files written and read by ModbusScope.
+
+## File structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)